### PR TITLE
fix(ecs): respect LogGroup's region for aws-log-driver

### LINKED ISSUE #17747: (ecs): AwsLogDriver ignores log group's region
### What is the problem?

ECS supports cross-region logging with awslogs with the `awslogs-region` option.
When I pass a cross-region `LogGroup` (constructed from an arn), the `awslogs-region` option in the resulting template is always the current region, ignoring the log group's region.

### Reproduction Steps

Create a cross-region log group and pass it to `ecs.AwsLogDriver`:

```python
log_group = logs.LogGroup.from_log_group_arn(f"arn:aws:logs:eu-west-2:{self.account}:log-group:my_log_group")

container = ecs.ContainerDefinition(self, "container", 
    ...
    logging=ecs.AwsLogDriver(stream_prefix="my_prefix", log_group=log_group)
)
```

### What did you expect to happen?

The logs are written to the specified log group.

### What actually happened?

The task execution role is given the correct permissions for the log group (including the correct region).

But the `awslogs-region` option in the container definition is always the same as the container's region.

Task fails to start.

### CDK CLI Version

1.134

### Framework Version

_No response_

### Node.js Version

17.1.0

### OS

Linux

### Language

Typescript, Python, .NET, Java, Go

### Language Version

_No response_

### Other information

I'm resorting to escape hatches to override `Properties.ContainerDefinitions.0.LogConfiguration.Options.awslogs-region` on the task definition, but this doesn't work if there are multiple containers.

### PR DESCRIPTION
**Pull request by kornicameister**
Respect passed log group `region` when constructing an instance of `AwsLogDriver`.
That change allows to implement cross region logging pattern for ECS containers.

fixes #17747

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by kornicameister** [comment]
cc @skinny85 @gshpychka

**Comment by kornicameister** [comment]
Alright, let's see if that one works now.

**Comment by kornicameister** [comment]
@madeline-k @skinny85 thanks to fix from another issue, this change now allows to have correct region when using imported log group. 
Cheers 👍

File: packages/@aws-cdk/aws-ecs/test/aws-log-driver.test.ts [hunk 1]
```diff
@@ -155,4 +155,41 @@ describe('aws log driver', () => {
...
   });
+
+  test('allows cross-region log group', () => {
+    // GIVEN
+    const logGroupRegion = 'asghard';
+    const logGroup = logs.LogGroup.fromLogGroupArn(
+      stack,
+      'LogGroup',
+      `arn:aws:logs:${logGroupRegion}:1234:log-group:my_log_group`,
+    );
```
Comment (madeline-k) [inline packages/@aws-cdk/aws-ecs/test/aws-log-driver.test.ts:166]:
```suggestion
    const logGroup = logs.LogGroup.fromLogGroupArn(stack, 'LogGroup',
      `arn:aws:logs:${logGroupRegion}:1234:log-group:my_log_group`);
```

**Review by madeline-k — CHANGES_REQUESTED** [review]
Thanks for contributing this fix, @kornicameister ! Looks great, I just have one tiny suggestion.

**Review by madeline-k — DISMISSED** [review]
Applied my small suggestion myself. Approving! Thanks, @kornicameister!

**Review by madeline-k — APPROVED** [review]

File: packages/@aws-cdk/aws-ecs/test/aws-log-driver.test.ts (hunk 1 above)
Comment (kornicameister) [reply packages/@aws-cdk/aws-ecs/test/aws-log-driver.test.ts]: Yeah, I noticed you prefer that style in cdk. I am more accustomed to Python and having everything spread across multiple lines. Makes it much easier to follow `git diff`

**Pull request merged**